src/xunit.analyzers/AssertEqualStringValueShouldNotUseGenericTypeFixer.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Immutable;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Editing;
+
+namespace Xunit.Analyzers
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp), Shared]
+    public class AssertEqualStringValueShouldNotUseGenericTypeFixer : CodeFixProvider
+    {
+        const string title = "Remove generic type";
```

Comment:
Technically you are removing the generic type argument, but the effect here is to use the string-specific Assert.Equal method overload. So maybe "Use string Assert.Equal"?